### PR TITLE
[Protocol3] Internal transfer improvements

### PR DESCRIPTION
**Pull request by Brechtpd**
@yueawang Because this repo's 'internal-transfer' branch is the branch used for the internal transfer   PR on the main protocols repo I'm making a PR here with my changes so when they get merged they will show up in the main PR.

Not much has changed really, mainly some renaming, fixed some small issues and added some tests. Please take a look!

### CODE REVIEW

File: packages/loopring_v3/test/testExchangeUtil.ts
Comment (yueawang) [inline packages/loopring_v3/test/testExchangeUtil.ts:3533]: I agree that we don't need to check online data consistency here, but for test purpose, should we keep that test function in maybe a new test suite to check if on-line data is valid?
Thanks!

**Review by yueawang — APPROVED** [review]

**Comment by yueawang** [comment]
Have we decided that we can merge the internal transfer circuit but do not enable until access control ready?
If so, we can merge it to master ASAP to avoid big divergence.

File: packages/loopring_v3/test/testExchangeUtil.ts
Comment (Brechtpd) [reply packages/loopring_v3/test/testExchangeUtil.ts:3533]: It's still checked in the tests! The test code now uses the explorer lib to get the on-chain data from Ethereum and reconstruct the Merkle tree. The actual code is now in `internal_transfer_processor.ts`.

**Comment by Brechtpd** [comment]
@yueawang I can't merge this PR, can you do it? Thanks!

I think we should merge ASAP like you said. Will ask Daniel after this PR is merged.

**Pull request merged**
